### ISSUE TITLE
ImportError: cannot import name 'match_histograms'

### ISSUE THREAD

**Issue by hhawary** [albumentations-team/albumentations#752]
## 🐛 Bug
After installing Albumentations using conda i get this error `ImportError: cannot import name 'match_histograms'` when i try to import the library

### 2. `conda install -c conda-forge imgaug`

Collecting package metadata (current_repodata.json): done
Solving environment: done

==> WARNING: A newer version of conda exists. <==
  current version: 4.8.3
  latest version: 4.9.2

Please update conda by running

`$ conda update -n base -c defaults conda`

\## Package Plan ##

  environment location: /home/<username>/anaconda2/envs/wmlce_env

  added / updated specs:
    - imgaug

The following packages will be downloaded:

```
    package                    |            build
    ---------------------------|-----------------
    ca-certificates-2020.11.8  |       h1084571_0         145 KB  conda-forge
    certifi-2020.11.8          |   py36h270354c_0         150 KB  conda-forge
    geos-3.8.0                 |       he6710b0_0         1.1 MB
    imgaug-0.4.0               |             py_1         832 KB  conda-forge
    python_abi-3.6             |          1_cp36m           4 KB  conda-forge
    shapely-1.7.1              |   py36h98ec03d_0         394 KB
    ------------------------------------------------------------
                                           Total:         2.6 MB

```
The following NEW packages will be INSTALLED:

  geos               pkgs/main/linux-ppc64le::geos-3.8.0-he6710b0_0
  imgaug             conda-forge/noarch::imgaug-0.4.0-py_1
  python_abi         conda-forge/linux-ppc64le::python_abi-3.6-1_cp36m
  shapely            pkgs/main/linux-ppc64le::shapely-1.7.1-py36h98ec03d_0

The following packages will be UPDATED:

  ca-certificates    pkgs/main::ca-certificates-2020.10.14~ --> conda-forge::ca-certificates-2020.11.8-h1084571_0
  certifi               pkgs/main::certifi-2020.6.20-py36_0 --> conda-forge::certifi-2020.11.8-py36h270354c_0

Proceed ([y]/n)? y

Downloading and Extracting Packages
```
python_abi-3.6       | 4 KB      | ################################################################################################################################################################ | 100%
shapely-1.7.1        | 394 KB    | ################################################################################################################################################################ | 100%
ca-certificates-2020 | 145 KB    | ################################################################################################################################################################ | 100%
imgaug-0.4.0         | 832 KB    | ################################################################################################################################################################ | 100%
certifi-2020.11.8    | 150 KB    | ################################################################################################################################################################ | 100%
geos-3.8.0           | 1.1 MB    | ################################################################################################################################################################ | 100%
```
Preparing transaction: done
Verifying transaction: done
Executing transaction: done

### 3. `conda install albumentations -c conda-forge`
Collecting package metadata (current_repodata.json): done
Solving environment: done

==> WARNING: A newer version of conda exists. <==
  current version: 4.8.3
  latest version: 4.9.2

Please update conda by running

    $ conda update -n base -c defaults conda

\## Package Plan ##

  environment location: /home/<username>/anaconda2/envs/wmlce_env

  added / updated specs:
    - albumentations

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    albumentations-0.5.1       |     pyhd8ed1ab_0          59 KB  conda-forge
    ------------------------------------------------------------
                                           Total:          59 KB

The following NEW packages will be INSTALLED:

  albumentations     conda-forge/noarch::albumentations-0.5.1-pyhd8ed1ab_0

Proceed ([y]/n)? y

Downloading and Extracting Packages
`albumentations-0.5.1 | 59 KB     | ################################################################################################################################################################ | 100%`
Preparing transaction: done
Verifying transaction: done
Executing transaction: done

`import albumentations as A`
## Expected behavior
```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-7c2496d7954d> in <module>
----> 1 import albumentations as A

~/anaconda2/envs/wmlce_env/lib/python3.6/site-packages/albumentations/__init__.py in <module>
      3 __version__ = "0.5.1"
      4
----> 5 from .core.composition import *
      6 from .core.transforms_interface import *
      7 from .core.serialization import *

~/anaconda2/envs/wmlce_env/lib/python3.6/site-packages/albumentations/core/composition.py in <module>
      6 import numpy as np
      7
----> 8 from albumentations.augmentations.keypoints_utils import KeypointsProcessor
      9 from albumentations.core.serialization import SerializableMeta
     10 from albumentations.core.six import add_metaclass

~/anaconda2/envs/wmlce_env/lib/python3.6/site-packages/albumentations/augmentations/__init__.py in <module>
      6
      7 # New transformations goes to individual files listed below
----> 8 from .domain_adaptation import *
      9
     10 from .utils import *

~/anaconda2/envs/wmlce_env/lib/python3.6/site-packages/albumentations/augmentations/domain_adaptation.py in <module>
      8 from .functional import clipped, preserve_shape
      9
---> 10 from skimage.exposure import match_histograms
     11
     12 from .utils import read_rgb_image

ImportError: cannot import name 'match_histograms'
```

## Environment

 - Albumentations version : 0.5.1 
 - Python version : Python 3.6.10
 - OS : Linux Redhat aivserver 4.14.0-115.6.1.el7a.ppc64le
 - How you installed albumentations : conda
 - Any other relevant information:

## Additional context
```
#Name                    Version                   Build                         Channel
scikit-image              0.15.0           py36he6710b0_0
scikit-learn              0.22.1           py36h22eb022_0
scipy                     1.3.1            py36he2b7bc3_0

```

**Comment by BloodAxe** [comment]
This function is available in `scikit-image` >= 0.18.
We should prehaps update https://github.com/albumentations-team/albumentations/blob/master/conda.recipe/meta.yaml with this dependency.

**Comment by hhawary** [comment]
> This function is available in `scikit-image` >= 0.18.
> We should prehaps update https://github.com/albumentations-team/albumentations/blob/master/conda.recipe/meta.yaml with this dependency.

I search for scikit-image >= 0.18 using IBM Anaconda powerai-release=1.6.2 but unfortunately i couldn't find even in their github [https://github.com/scikit-image/scikit-image](url). I check anther PC that have scikit-image = 0.16.2 and it work fine.

The IBM server I'm using is incomparable with scikit-image = 0.16.2 due to conflicts in packages.

Anyway i found that match_histograms was in transforms folder back in version 0.15 so what i did is changing this line:
```
~/anaconda2/envs/wmlce_env/lib/python3.6/site-packages/albumentations/augmentations/domain_adaptation.py in <module>
      8 from .functional import clipped, preserve_shape
      9
---> 10 from skimage.exposure import match_histograms
     11
     12 from .utils import read_rgb_image
```

to be

`from skimage.transform import histogram_matching`

and that solve the problem of importing `import albumentations as A.` i don't know if there will be any other problem due related to this change in other functions or not since i didn't check.

@BloodAxe thanks.
